Update README to reflect run/update commands

jayeff opened the pull request:
Current README is slightly outdated. I updated it to use the `run` and `update` command where appropriate

dee-me-tree-or-love commented:
Hey @jayeff, this is awesome! Thanks a lot, I will take a good look later today :) ✌️

jayeff commented:
@dee-me-tree-or-love JFYI: I rebased onto latest master and updated commit message to match style in CHANGES.rst (i.e. included `DOC:` type at beginning). No other changes to README were applied from my original commits.

dee-me-tree-or-love commented:
Hey @jayeff ! Thanks a lot for the update and sorry for taking longer than expected to review. I'm getting to this shortly and will keep you updated :v:

jayeff commented:
@dee-me-tree-or-love Don't worry. One own life is more important than taking care of some random GitHub PRs and issues 😊

btw. pypy3.9 test run did fail. I assume not related to my README change 😉

dee-me-tree-or-love commented:
Hey @jayeff ! Thanks a lot for your understanding, indeed, indeed, this helps to stay afloat, appreciated :)

> btw. pypy3.9 test run did fail. I assume not related to my README change 😉

Oh yes! Of course not, it's indeed a known limitation, I still didn't get to catching the root cause of these failures, but they happen every now and then. There's a related issue about them: #408 . Sorry for the inconvenience, I really hope to get to the bottom of this soon :) Usually a restart just resolves the failing run.

dee-me-tree-or-love reviewed (changes requested):
hey @jayeff ! Thanks a lot for the work here and for keeping an eye out for detail. Sorry for such a delay in reviewing :) I think this is generally ready to get merged. However, I've left a few small suggestions, so if you have a moment to take a look, I'd really appreciate it. Plus, we've released `1.5.1` in the meantime and also added a small change with changes in the README (#442), if you could solve the merge issues, I'll be ready to take a look again ✌️

dee-me-tree-or-love commented:
Hey @jayeff, just wanted to give a friendly bump here :) I'd really like to merge these changes, so I wonder if you may have the chance to give the comments a look sometime soon? :) Otherwise, if you don't object, do you mind if I will resolve the conflicting files, merge the PR and do adjustments from there? Let me know! If you ain't got time for this, I'll be happy to pick it up further :)

jayeff commented on README.rst:
```diff
@@ -381,7 +323,7 @@ To serve packages from a directory on the host, e.g. ``~/packages``::
 To authenticate against a local ``.htpasswd`` file::
 
-    docker run -p 80:8080 -v ~/.htpasswd:/data/.htpasswd pypiserver/pypiserver:latest -P .htpasswd packages
+    docker run -p 80:8080 -v ~/.htpasswd:/data/.htpasswd pypiserver/pypiserver:latest run -P .htpasswd packages
```
I updated the lines you mentioned plus some more I missed on my first pass (see latest commit on this PR)

jayeff commented on README.rst:
```diff
@@ -416,28 +358,21 @@ following command, assuming you have *git* installed on your ``PATH``::
-
-https://github.com/dexterous/pypiserver-on-the-cloud contains
-instructions on how to run ``pypiserver`` on one of the supported cloud
-service providers.
```
I still argue that the `Running on Heroku/Dotcloud` section and link to this repo should be removed.

- dotcloud is [defunct since 2016](https://www.failory.com/cemetery/dotcloud)
- Heroku is still available but they did have a [bad security incident last year](https://www.theregister.com/2022/05/04/heroku_security_communication_dubbed_complete/) (plus handled it quite badly too) plus they did [shut down free dynos by end of last year](https://help.heroku.com/RSBRUH58/removal-of-heroku-free-product-plans-faq) making it less useful to recommend

When searching [GitHub repos for pypiserver](https://github.com/search?o=desc&q=pypiserver&s=updated&type=Repositories) you can find more recent projects on how to run pypiserver. e.g.:

- how to run pypiserver with docker https://github.com/harryzcy/docker-pypiserver
- a docker compose version https://github.com/joepreludian/pypiserver/
- a rudimentary helm-chart https://github.com/mittwald/pypiserver-helm
- yours for running it on Google AppEngine ;-) https://github.com/deemetree-at-satelligence/pypiserver-appengine

dee-me-tree-or-love replied:
Right, true, I do agree with this argument, thanks a lot for the clarification. That's very fair points, so let's indeed remove it, if anyone is asking for it later, just more the reason to provide more up to date guides 👍 😊  Great stuff, thanks for the discussion 👍

jayeff commented on README.rst:
```diff
@@ -90,123 +90,65 @@ not officially supported, and will not receive bugfixes or new features.
 
    See also `Client-side configurations`_ for avoiding tedious typing.
 
-4. Enter ``pypi-server -h`` in the cmd-line to print a detailed usage message::
```
I reverted this to current output for `pypi-server -h` 👍

jayeff commented:
@dee-me-tree-or-love Thanks for the reminder. I forgot a bit about this PR 😊 

I addressed your feedback, resolved the merge conflict and rebased on current master.

dee-me-tree-or-love commented:
> @dee-me-tree-or-love Thanks for the reminder. I forgot a bit about this PR 😊
> 
> I addressed your feedback (see individual comments to your PR review), resolved the merge conflict and rebased on current master.
> 
> **Update** Also did fix CI status badge 😎

Hey @jayeff! Thanks a lot for the update, very happy to hear from you and see the changes! I'll review this shortly today, I think it looks great, so I'm pretty sure I'll merge it in the coming days 😊 👍

dee-me-tree-or-love reviewed (approved):
@jayeff thanks a lot for the great work! This is looking very good, and I'm really excited about an updated README, that was about time :D I'm approving this, but there are some CI/CD issues that are not related to this PR that I will fix first, and then merge this 🙆

dee-me-tree-or-love commented:
Hey @jayeff, thanks again for the updates, merging this now since the work in #474 and #471 (cc kudos to @cclauss) has resolved the CI issues 👍
